Scripts/datastorage.js

```diff
@@ -1,57 +1,25 @@
 'use strict';
+
```

Comment:
The reason you have 622 files and +97,834 admissions is because you pushed up your node_modules. We do not want those there. By using a .gitignore, this will solve that problem :)